### ISSUE TITLE
Cloud cover forecast is often over-optimistic in WRF

### ISSUE THREAD

**Issue by tcibils** [soaringmeteo/soaringmeteo#85]
I might be wrong, but I think I noticed that high-altitude clouds (8km+) are not taken into consideration in the soaringmeteo forecasts? Is that correct? As they limit sunshine and thermals a lot, and could hence be a valuable part of the forecast. I currently use meteoblue cloud forecasts to check that.

**Comment by tcibils** [comment]
Example from yesterday:

![2023-06-18-soaring](https://github.com/soaringmeteo/soaringmeteo/assets/18047049/68b97e30-23b5-4641-9916-0f81387fb00f)

![2023-06-18-webcam-naye](https://github.com/soaringmeteo/soaringmeteo/assets/18047049/e8de91a8-1a33-4872-bd0c-d7867d8f9c8c)

**Comment by julienrf** [comment]
In soarWRF, we take into account the incoming sun radiations. In your example, the 91% at 14:00 means that 9% of the incoming radiation flux is blocked by the cloud cover. If you open the meteogram, you will see that at 15:00 the level of insolation is only 56%.

So, although the cirrus clouds are not shown visually in the meteograms, we do take into account the incoming radiation, which is impacted by the cloud cover.

That being said, it is possible that the amount of incoming radiation is not correctly computed (I also often noticed that soarWRF was pretty optimistic about the levels of insolation). This is something we need to check.

Note that in soarV2 we do show the cirrus clouds in the meteograms:

![image](https://github.com/soaringmeteo/soaringmeteo/assets/332812/bb713701-d548-4174-8432-0ffeec7c9760)

(note that the GFS model for that day predicted a cloud cover of 51% in that area)

As soon as we will switch to the soarV2 interface for the WRF forecast, the cirrus clouds will be shown visually in the WRF meteograms.

**Comment by julienrf** [comment]
I think we can close this issue now that the WRF results are shown in the v2 interface:

![image](https://github.com/soaringmeteo/soaringmeteo/assets/332812/ecfcd27e-29b5-46b7-823e-9079e985765f)

Now, the forecasted cloud cover is shown in the meteograms.

It is possible that our WRF model could be fine-tuned to increase the accuracy of the cloud cover forecast, but that should be a separate issue.
